README.md

```diff
@@ -91,37 +91,68 @@ $apiResponse = $request->changeAvailability(new ContentAvailability('product/url
 
 ---
 In all request you can catch some exceptions:
-* `ToManyItemsException` - make request with less items,
+* `BadRequestException` - bad request,
+* `ToManyItemsException` - make request with fewer items,
```

Comment:
`TooMany` - is this typo just here or in the exception as well?

Reply:
Everywhere :+1: